src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -437,6 +488,17 @@ public String getAsLastNames(boolean oxfordComma) {
         return authorsLastOnly[abbrInt];
     }
 
+    public String getAsLastNamesLatexFree(boolean oxfordComma) {
+        int abbrInt = oxfordComma ? 0 : 1;
```

Comment:
We normally try to avoid abbreviations. How ironic... :wink:
But I can see that this particular abbreviation was used before in other places in this class too, so never mind.
If you like, change it in all places in this class, if not, we can probably merge.

Reply:
I left one abbreviation because I am not really sure what it should be (firstabbr)

Reply:
If in doubt, just name it firstAbbreviation 😆

Reply:
> If in doubt, just name it firstAbbreviation 😆

😱

Joke aside. firstAbbreviation is actually an improvement on firstabbr, fair enough 😝